src/parser.rs

```diff
@@ -3898,6 +3898,12 @@ pub fn parse_call(tokens: &[Token], index: &mut usize) -> (Call, Option<JaktErro
                             index,
                             ExpressionKind::ExpressionWithoutAssignment,
                         );
+
+                        if let Some(jakt_error) = &err {
```

Comment:
How about checking if the index is changed instead?

Reply:
Well, it might be perfectly fine to not increment the index sometimes? Either way I think it's more generic to handle the sudden appearance of an error, no matter the reason.